packages/tools/src/tools/ZoomTool.ts

```diff
@@ -131,6 +147,55 @@ class ZoomTool extends BaseTool {
     viewport.render();
   }
 
+  _dragCallbackVideoViewport(evt) {
```

Comment:
Ok this is exactly what i was looking for a map between ICamera and IVideoCamera, if you put this in the setCamera you have half of it done